packages/firestore/test/integration/api/transactions.test.ts

```diff
@@ -523,6 +524,64 @@ apiDescribe('Database transactions', persistence => {
             'Firestore transactions require all reads to be executed'
           );
         });
+
+      const postSnap = await getDoc(docRef);
+      expect(postSnap.get('foo')).to.equal('baz');
+    });
+  });
+
+  it('cannot read after writing and does not commit - get is awaited', () => {
```

Comment:
This test is exactly the same as the one above

Reply:
Thanks. You're right. I'm removing this test because I can't think of how I could make a meaningful change to the test based on this description.